lib/YaST/RestartInfoPage.pm

```diff
@@ -19,6 +19,7 @@ sub init {
 
 sub get_restart_info_page {
     my ($self) = @_;
+    $self->wait_restart_info_popup();
```

Comment:
this call needs to go in the test layer, this getter should have only one purpose not two.

Reply:
Updated.